data/transition-sites/esfa_ncs.yml

```
@@ -0,0 +1,8 @@
+---
+site: esfa_ncs
```

```suggestion
site: esfa_nationalcareers
```

I think `ncs` might not be immediately obvious to people if they're looking at this for the first time.